mapper-processor/src/test/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityPropertyAnnotationsTest.java

```diff
@@ -105,7 +105,7 @@ public void should_fail_with_expected_error(String expectedError, TypeSpec entit
   public static Object[][] entitiesWithErrors() {
     return new Object[][] {
       {
-        "Properties can't be annotated with both @PartitionKey and @ClusteringColumn.",
+        "Properties can't be annotated with both @ClusteringColumn and @PartitionKey.",
```

Comment:
Was wondering if we had a way to test for compiler errors being surfaced, I should add some more tests as I had been testing manually up to this point, but if it expedites things more I can add tests when i work on [JAVA-2111](https://datastax-oss.atlassian.net/browse/JAVA-2111)

Reply:
See `EntityPropertyAnnotationsTest`.

Those tests are pretty slow, so try not to add too many (I'm thinking of adding something in the POM to skip them).

Reply:
Mmm forget that, looks like I misunderstood your question.

Reply:
No need to expedite this PR, we might as well add all necessary tests now.